### ISSUE TITLE
Time Signature Toolbar is not accessible

### ISSUE THREAD

**Issue by SteveDaulton** [audacity/audacity#4983]
### Bug description

It is impossible to use the Time Signature Toolbar with keyboard commands.

### Steps to reproduce

Try using the Time Signature Toolbar without a mouse / pointing device.

### Expected behavior

To be able to tab into and out of the Time Signature Toolbar

### Audacity Version

current master / alpha / nightly build

### Operating system

Linux, macOS

### Additional context

No such issue on Windows

**Comment by SteveDaulton** [comment]
In the time controls in the Selection Toolbar, you move between hh <-> mm <-> ss with cursor keys.

In the Time Signature Toolbar, I have not found any way, cursor keys, tab, Enter, or otherwise, to move between the Tempo field and the time signature fields.

Also, on my machine, the "note value" number only shows one digit, so when 16th notes are selected, it looks like this:
![ts](https://github.com/audacity/audacity/assets/11681097/4bf2d487-041a-4b52-a87e-ceb277c489c9)

**Comment by LWinterberg** [comment]
the time control is one element, the time signature toolbar contains 3 separate elements, namely 2 text/spin controls and one dropdown. A better comparison would be the selection toolbar, which contains 2 time controls and one button - switching between the two also requires tab. Which also is accessible.

**Comment by SteveDaulton** [comment]
> A better comparison would be the selection toolbar, which contains 2 time controls and one button - switching between the two also requires tab. Which also is accessible.

I can tab into the tempo box, but then focus is stuck there, requiring a mouse click to get into the time signature boxes.

**Comment by LWinterberg** [comment]
I also can tab out of the tempo box.

**Comment by crsib** [comment]
I'm pretty sure that this issue is Linux specific

**Comment by DavidBailes** [comment]
I can confirm that on XUbuntu 22.04, if you Tab to the Tempo control, then the focus gets stuck there - which isn't good news for keyboard users :(

**Comment by DavidBailes** [comment]
Checked using a friend's iMac , running MacOS 10.15.7
There is a similar, but not identical issue: The focus gets stuck in the Tempo control. However you can still move to the upper toolbars or the tracks using Cmd+F6. But once stuck in Tempo, there's no way to move to the other controls on the lower toolbars.

**Comment by DavidBailes** [comment]
I've removed the linux label, as MacOS is also affected.

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
